src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    public Circle(int radius, String color) {
```

Comment:
Call the parent class constructor with the 'color' parameter, instead of setting the color field in Circle class. Use 'super(color);' to achieve this. 🚀